service/src/main/java/org/apache/griffin/core/metastore/hive/HiveMetaStoreServiceJdbcImpl.java

```diff
@@ -160,6 +143,22 @@ public void evictHiveCache() {
         LOGGER.info("Evict hive cache");
     }
 
+    private void closeConnection(Connection conn, Statement stmt, ResultSet rs) {
+        try {
+            if (rs != null) {
```

Comment:
I see these three resources are AutoCloseable. please use try-with-resource. don't need to close them in finally block.
https://docs.oracle.com/javase/tutorial/essential/exceptions/tryResourceClose.html

Reply:
In current version I used the DriverManager to get Hive Connection, if changed to try-with-resource, I cannot do the unit test with Mockito. So we keep try-catch-finally for now, and if we will change to use Hive Connection Pool, we may change to try-with-resource method.

Reply:
> In current version I used the DriverManager to get Hive Connection, if changed to try-with-resource, I cannot do the unit test with Mockito. So we keep try-catch-finally for now, and if we will change to use Hive Connection Pool, we may change to try-with-resource method.

I also suggest you to use try-with-resource. For test, I think there is a way to do it .